ci-operator/step-registry/cucushift/upgrade/eus/pause-worker-mcp/cucushift-upgrade-eus-pause-worker-mcp-commands.sh

```diff
@@ -34,4 +40,32 @@ then
     source "${SHARED_DIR}/proxy-conf.sh"
 fi
 
-pause
+# check all actual mcp, if some of them unknown then break the job.
+expected_mcp=("master" "worker")
+custom_mcp_num=$(echo "${MCO_CONF_DAY2_CUSTOM_MCP}" | jq -r ". | length")
+last_mcp_index=$((custom_mcp_num - 1))
+for index in $(seq 0 ${last_mcp_index}); do
+    custom_mcp_name=$(echo "${MCO_CONF_DAY2_CUSTOM_MCP}" | jq -r ".[$index].mcp_name")
+    if [[ -z "$custom_mcp_name" ]]; then
+        echo "No mcp_name input"
+        exit 255
+    fi
+    expected_mcp+=("$custom_mcp_name")
```

Comment:
Actually we do need the complex loop to get `expected_mcp`, we can use the following line to get it easily.
```
expected_mcp+=($(echo "${MCO_CONF_DAY2_CUSTOM_MCP}"|  jq -r '.[].mcp_name'))
```